### DISCUSSION TITLE
scenes's floor missing

### DISCUSSION THREAD

**Discussion by m061i6** [mozilla/hubs#5401] [Hubs Cloud Questions] [answered]
hello,
we use your product Hubs Cloud AWS and build an app.
but now we get a trouble.

we import our scenes in the app backend.
it succese but when we open a room with this scenes
the scenes's floor is missing(like image)

but we published on hub it is no problem.

here is the import url(no problem):https://hubs.mozilla.com/scenes/E5axXTg
here is our scenes(missing floor):https://godpod.world/scenes/oGkSOOi

I would be grateful for an early reply.
Thank you.

(scenes missing floor)
![image](https://user-images.githubusercontent.com/13673776/165880183-c9adfb4b-a822-4f89-9f1f-ec0eaf7ac509.png)

**Comment by matthewbcool** [comment, marked as answer]
Looks like there are some issues with the glb itself.  https://discord.com/channels/498741086295031808/970288550035742720/970306729143324753
